<?xml version="1.0"?>
<launch>
  <!-- Launch file parameters -->
  <arg name="debug"     default="true" />
  <arg name="effort_interface"     default="false" />

  <arg if=      "$(arg debug)"  name="DEBUG" value="screen"/>
  <arg unless = "$(arg debug)"  name="DEBUG" value="log"/>

  <!-- Controllers config -->
  <rosparam file="$(find ur_control)/config/ur3e_controllers.yaml" command="load" />
  <rosparam file="$(find ur_control)/config/gripper_controller_osx.yaml" command="load" />

  <!-- Load controllers -->
  <node name="robot_controllers" pkg="controller_manager" type="spawner" respawn="false"
        output="$(arg DEBUG)"
        args="joint_state_controller scaled_pos_joint_traj_controller gripper_controller"/>

  <!-- TF -->
  <node pkg="robot_state_publisher" type="state_publisher" name="state_publisher"> 
    <param name="publish_frequency"  type="double" value="500" />
  </node>
</launch>
